packages/react-devtools-core/src/backend.js

```diff
@@ -355,6 +373,7 @@ export function connectWithCustomMessagingProtocol({
   }
 
   const agent = new Agent(bridge);
+  bridge.send('isReloadAndProfileSupported', isReloadAndProfileSupported());
```

Comment:
I think we should move it somewhere. Can this actually be an argument on `initBackend`?

Notice that you don't have any changes related to the browser extension, this is because it has its own logic for backend initialization:
https://github.com/facebook/react/blob/fc4a33eaa9c935ac860ab6043b95d55540068571/packages/react-devtools-extensions/src/contentScripts/backendManager.js#L99-L161

Then, inside `initBackend`, we could do something like:
```
if (isReloadAndProfileSupported()) {
  agent.onReloadAndProfileSupportedByHost();
}
```

and inside `Agent` this method would just send this event:
```
this._bridge.send('isReloadAndProfileSupported', true);
```

and the only remaining thing is hardcoding `true` for `initBackend` call in the extension code, which I've referenced above

Reply:
This will allow us to reuse as much as possible, and reduce the fragmentation: instead of having custom logic, like here in `react-devtools-core/src/backend.js`, we would pass an argument to `initBackend`, which is from `react-devtools-shared`

Reply:
Great idea! Much cleaner to have this logic back into `Agent`/`initBackend`. Updated